src/dao/ERC20ManagerDAO.js

```diff
@@ -122,50 +122,19 @@ export default class ERC20ManagerDAO extends AbstractContractDAO {
       })
       map = map.set(ethToken.id(), ethToken)
 
-      if (btcDAO.isInitialized()) {
-        try {
-          const { balance, balance0, balance6 } = await btcDAO.getAccountBalances()
-          const btcToken = new TokenModel({
-            dao: btcDAO,
-            name: btcDAO.getName(),
-            symbol: btcDAO.getSymbol(),
-            isApproveRequired: false,
-            balance,
-            balance0,
-            balance6,
-            isOptional: false,
-            isFetched: true,
-            blockchain: 'Bitcoin',
-          })
-          map = map.set(btcToken.id(), btcToken)
-        } catch (e) {
-          // eslint-disable-next-line
-          console.log('BTC support is not available', e)
-        }
-      }
-
-      if (bccDAO.isInitialized()) {
-        try {
-          const { balance, balance0, balance6 } = await bccDAO.getAccountBalances()
-          const bccToken = new TokenModel({
-            dao: bccDAO,
-            name: bccDAO.getName(),
-            symbol: bccDAO.getSymbol(),
-            isApproveRequired: false,
-            balance,
-            balance0,
-            balance6,
-            isOptional: false,
-            isFetched: true,
-            blockchain: 'Bitcoin Cash',
-          })
-          map = map.set(bccToken.id(), bccToken)
-        } catch (e) {
-          // eslint-disable-next-line
-          console.log('BCC support is not available', e)
+      const bitcoinLikeTokens = await Promise.all([
```

Comment:
is bitcoin erc20-token?

Reply:
See previous comment and the link attached.